control/tests/test_heal.py

```diff
@@ -1,36 +1,104 @@
-
+"""
+Test heal processor
+"""
+import datetime
+import json
 import os
 import sys
-import unittest
+from django.test import TestCase
+from datetime import timedelta
 
-"""
-Test heal processor
+import responses
+from django.utils.timezone import utc
+from mock import PropertyMock, patch
+
+from control.veda_heal import VedaHeal
+from VEDA_OS01.models import URL, Course, Destination, Encode, Video
+from VEDA_OS01.utils import build_url
 
-"""
 sys.path.append(os.path.dirname(os.path.dirname(
     os.path.abspath(__file__)
 )))
-from control.veda_heal import VedaHeal
-from VEDA_OS01.models import URL, Video, Encode
+
+CONFIG_DATA = {
+    'heal_start': 2,
+    'heal_end': 50,
+    'val_api_url': 'http://val.edx.org/api',
+    'val_token_url': 'http://val.edx.org/token',
+    'val_username': 'val_username',
+    'val_password': 'val_password',
+    'val_client_id': 'val_client_id',
+    'val_secret_key': 'val_secret_key',
+    'val_profile_dict': {
+        'desktop_mp4': ['desktop_mp4'],
+        'hls': ['hls'],
+        'mobile_low': ['mobile_low']
+    }
+}
```

Comment:
This would be great but I am not sure what is the best strategy to handle this? I can think of two ways:
1. Create a new config file(`test_instance_config.yaml`) for tests and then update all the occurrences of `instance_config.yaml` and based on a `settings`, decide which config file we need to load.
2. Update the `instance_config.yaml` with sane default values. Note: My understanding is that for production `instance_config.yaml` is replaced with production settings so there will be no issue in keeping default values.